src/frontend/components/UI/SearchBar/index.tsx

```diff
@@ -30,19 +35,19 @@ export default React.memo(function SearchBar() {
   const list = useMemo(() => {
     // Set can't handle spread of undefined. Leading to
```

Comment:
I think you can remove these comments now.

Reply:
done!